nautobot/core/admin.py

```diff
@@ -48,3 +50,9 @@ class ConfigAdmin(ConstanceAdmin):
 
 admin_site.unregister([Config])
 admin_site.register([Config], ConfigAdmin)
+
+
+class BaseAdmin(ModelAdmin):
```

Comment:
`BaseAdmin` is a pretty generic name. Perhaps `NautobotModelAdmin` would be better?